### ISSUE TITLE
Camillo Jr. fix

### ISSUE THREAD

**Issue by Danial29** [UnderscoreKilburn/repentance-issues#939]
You know how bullshit Camillo Jrs can be, so i suggest these changes. I think Camillo Jr. can be a great enemy with these fixes
1. Longer animation
2. Telegraphs
3. Only one can attack at a time
You can do all or choose one of them, that's your choice, but I think 2 and 3 are especially important so they definitely need to be in the game.
Video-example

https://user-images.githubusercontent.com/95134927/189543091-9204c3ab-f8e8-45fd-98fc-c9994ccc2a24.mp4

**Comment by krispypants** [comment]
I approve, this is probably the best fix for this enemy we could ask for, they are still tough but now they are pretty fair and fun to dodge now

**Comment by pelzz** [comment]
Yeah, this is a great suggestion, these enemies currently are very unpredictable at times and will hit you without much reaction, it gets very sickening, this would definitely make them way more fair.

**Comment by GentleSin37** [comment]
I approve the hell out of this. Kilburn please take this into account when reworking the little shit.

**Comment by chaathon** [comment]
please Camillos are actually painful to deal with i fully support this
